ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,360 @@
+1613.4 "Disgorge" #sync / 1[56]:[^:]*:Kokytos:8124:/
+1625.7 "Ravening (Fighter)" #sync / 1[56]:[^:]*:Kokytos:8119:/
+
+
+
+1800.0 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/
+1800.9 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8142:/
+1802.2 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8124:/
+1806.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+1809.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+# Fighter phase 2
+1814.5 "Ravening (Fighter)" sync / 1[56]:[^:]*:Kokytos:8119:/
+1824.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1832.7 "Archaic Rockbreaker" sync / 1[56]:[^:]*:Kokytos:8128:/
+1834.3 "Shockwave" sync / 1[56]:[^:]*:Kokytos:8129:/
+1837.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1840.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1841.2 "Archaic Rockbreaker 1" sync / 1[56]:[^:]*:Kokytos:812A:/
+1843.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1846.2 "Archaic Rockbreaker 2" sync / 1[56]:[^:]*:Kokytos:812A:/
+1851.3 "Front Combination/Rear Combination" sync / 1[56]:[^:]*:Kokytos:814[79]:/
+1851.5 "Inside Roundhouse" sync / 1[56]:[^:]*:Kokytos:812B:/
+1854.5 "Swinging Kick" sync / 1[56]:[^:]*:Kokytos:812[DE]:/
+1862.6 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8116:/
+1863.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8117:/
+1864.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1867.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+1874.9 "Ascendant Fist" sync / 1[56]:[^:]*:Kokytos:8131:/
+1877.0 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8125:/
+1881.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+1884.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+1887.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+
+
+# Behemoth phase 2
+1892.0 "Ravening (Behemoth)" sync / 1[56]:[^:]*:Kokytos:811A:/
+1902.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+1905.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+1915.6 "Comet" sync / 1[56]:[^:]*:Kokytos:8134:/
+1916.6 "--sync--" sync / 1[56]:[^:]*:Comet:8135:/
+1924.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:813B:/
+1925.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:813C:/
+1926.6 "Ecliptic Meteor" sync / 1[56]:[^:]*:Kokytos:8188:/
+1926.7 "--sync--" sync / 1[56]:[^:]*:Comet:8143:/
+1930.7 "Beastly Bile" sync / 1[56]:[^:]*:Kokytos:813D:/
+1932.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:813E:/
+1932.6 "Burst" sync / 1[56]:[^:]*:Comet:8136:/
+1939.8 "Pulverizing Pounce" sync / 1[56]:[^:]*:Kokytos:813F:/
+1943.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+1953.2 "Gluttonous Rampage" sync / 1[56]:[^:]*:Kokytos:8139:/
+1953.4 "Ravenous Bite 1" sync / 1[56]:[^:]*:Kokytos:813A:/
+1955.6 "Ravenous Bite 2" sync / 1[56]:[^:]*:Kokytos:813A:/
+1957.6 "Touchdown" sync / 1[56]:[^:]*:Kokytos:8137:/
+1958.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8146:/
+1960.8 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8126:/
+1968.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+1971.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+
+
+# Mage phase 3
+1979.3 "Ravening (Mage)" sync / 1[56]:[^:]*:Kokytos:8118:/
```

Comment:
This is as far as my longest log goes, but in Duty Finder in my experience the boss will usually die in the second Mage or Fighter phase, I rarely see the 2nd Behemoth phase.